### DISCUSSION TITLE
Clone SONOFF Zigbee Bridge ?

### DISCUSSION THREAD

**Discussion by JoeWiseman** [arendst/Tasmota#17754] [Q&A] [answered]
Hi there,

allthough it is rather boring job to teach devices to the bridge an the bridge is rather cheep, I was wondering, if it is possible to clone a bridge, so in case of a defect, I just can connect the backup-bridge to keep the system running with minimal delay.

Though I'm not really familiar with the Zigbee-protocol I thought about asking you as the experts.

Would be great to get some hints.

Best regards
Joe

**Comment by s-hadinger** [comment, marked as answer]
This is generally not doable. The Zigbee MCU needs to have the devices in its own tables and the only way to do it is to pair again all devices.

**Reply by JoeWiseman** [reply]
Hi,
thanks a lot for giving the answer I was searching for .... allthough I was hoping for a lucky ending ....
Best regards
Joe
